test/functional/apps/discover/_date_nanos.js

```diff
@@ -21,21 +21,24 @@ import expect from '@kbn/expect';
 
 export default function({ getService, getPageObjects }) {
   const esArchiver = getService('esArchiver');
-  const PageObjects = getPageObjects(['common', 'timePicker', 'discover']);
+  const PageObjects = getPageObjects(['common', 'timePicker', 'context', 'discover']);
```

Comment:
looks like `context` was added here but isn't used.

Reply:
`context` being used  in the test. Without it, test will fail.